Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/LandscapeIntegrationTest.java

```diff
@@ -129,6 +141,7 @@ public void testTools() {
 			onView(withId(R.id.pocketpaint_layout_tool_specific_options))
 					.check(matches(not(isDisplayed())));
 		}
+		IdlingRegistry.getInstance().unregister(idlingResource);
```

Comment:
You already invoked .unregister() in the tearDown method, so I think there is no need to additionally .register() at the start of the test.